src/main/java/com/manardenza/controller/HotelController.java

```diff
@@ -0,0 +1,50 @@
+package com.manardenza.controller;
+
+import com.manardenza.entity.Hotel;
+import com.manardenza.entity.Room;
+import com.manardenza.entity.User;
+import com.manardenza.servise.HotelService;
+
+import java.util.List;
+import java.util.Map;
+
+public class HotelController {
+
+    private static HotelController instance;
+    private HotelService hotelService;
+
+    private HotelController() {
+        hotelService = HotelService.getInstance();
+    }
+
+    public static HotelController getInstance() {
+        if (instance == null) {
+            instance = new HotelController();
+        }
+        return instance;
+    }
+
+    public void userRegistration(User userToRegistration) {
```

Comment:
1. better to call methods as actions, e.g., registerUser
2. Consider moving this and all user related methods to UserController + UserService